packages/zoe/test/unitTests/contracts/newSwap/test-newSwap-swap.js

```diff
@@ -580,6 +587,23 @@ test('multipoolAutoSwap doubleSwap', async t => {
   t.deepEqual(await E(publicFacet).getProtocolPoolBalance(), {
     RUN: runningFees,
   });
+
+  const collectFeesInvitation = await E(
```

Comment:
You don't need to `await` the invitation.

Reply:
done.  Probably copied from another example.

Under `zoe/test`, I see 84  matches for "invitation = await ".  more than 30 are "invitation = await .*.claim", which is probably alright, but another 30+ are "invitation = await .*make", which is the same as this. I'll go back and clean them up later.